scripts/legacy-kvm-vfat-boot.yaml

```diff
@@ -0,0 +1,47 @@
+#clear-linux-config
+
+# switch between aliases if you want to install to an actuall block device
```

Comment:
Same here

Reply:
Also capitalize switch and end with period to match the typo pull request.

Reply:
Done. Thanks.